graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -75,14 +90,19 @@ def rename_schema(ast, renamings):
 
     # Rename query type fields
     ast = _rename_query_type_fields(ast, renamings, query_type)
+
+    # check for fields or unions that depend on types that were suppressed
```

Comment:
Please try to keep code style in line with the surrounding code. Here, the comment a few lines above is capitalized, but your newly-added one is not. This particular instance is very minor, but every little bit counts and we want to keep the codebase as clean and polished-looking as possible.